Week2/homework/index.js

```diff
@@ -0,0 +1,114 @@
+console.log(1);
```

Comment:
Can you split the homework up in separate files:)?

> For all the following exercises create a new .js file. Try to find a proper name for each file or make a small comment about what it does inside for future reference.

Reply:
Hi,
Thank you for the enlightening.
I made a new .js files.
i gave them explanatory names.
and wrote short comments for each of the questions. 

Regards